README.md

```diff
@@ -41,8 +41,9 @@ A comprehensive manual of Fix8 GUI and every feature can be found here: [Fix8_Ma
     Download Mac app here: <a id="raw-url" href="https://github.com/nalmadi/fix8/releases/download/v1.3/fix8_v1.3_Mac.zip">[Mac App]</a>
     
     >- Right-click on `fix8` and select `open.`
-    >- A message that reads `fix8.app can't be opend because Apple cannot check it for malicious software.` click on open to run Fix8.
-    >- First run takes a couple of minutes for the files to be scanned, please be patient.  Running the app after that will take less than five seconds.
+    >- A message that reads `fix8.app can't be opened because Apple cannot check it for malicious software.` Go to security preferences & click on open to run Fix8.
```

Comment:
This is great, thanks @najam-tariq!  Can you share a screenshot or a short clip of the security preferences walk through.  This is the one I have: 
![image](https://github.com/user-attachments/assets/91be1745-5837-43eb-97ef-173ba2a97ea0)

Reply:
- Here are multiple screenshots based on what I see: 

1) 
<img width="275" alt="Screenshot 2024-12-24 at 8 39 23 PM" src="https://github.com/user-attachments/assets/39702733-a022-4533-96a5-47e0d59c849d" />

2)
<img width="481" alt="Screenshot 2024-12-24 at 8 39 50 PM" src="https://github.com/user-attachments/assets/556d492b-a4b9-4cda-967a-83122d2f0011" />

3)
<img width="254" alt="Screenshot 2024-12-24 at 8 39 59 PM" src="https://github.com/user-attachments/assets/ce6ca806-571b-4f29-92ee-4ba4f71a746b" />

Reply:
Thanks @najam-tariq!  Do you think it would be best to create a separate Markdown for Mac download and installation, including these screenshots and possibly other troubleshooting ideas?  Or should we add that to the manual and just rely on the manual to keep the readme file short?